src/migration/1710417425669-UserPersonalizedDigestSendType.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserPersonalizedDigestSendType1710417425669 implements MigrationInterface {
+    name = 'UserPersonalizedDigestSendType1710417425669'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "user_personalized_digest" ADD "flags" jsonb NOT NULL DEFAULT '{}'`);
+        await queryRunner.query(`CREATE INDEX "IDX_user_personalized_digest_flags_sendType" ON "user_personalized_digest" ((flags->>'sendType'))`);
```

Comment:
Works, but wondering if you want to set everyone to weekly now in migration?
(Not relevant now, but lets say we introduce a third one)
Alternative would be default setting

Reply:
I am looking at sendType as something optional and tied to experiment, so I don't want defaults until we see how it will go and what actual send types we will have in the future or will we even have them. Does that makes sense?